bin/set-up-vs-code.js

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env node
+// @ts-check
+
+/**
+ * Links or unlinks `.vscode/settings-sample.jsonc` to `.vscode/settings.json`.
+ *
+ * Usage:
+ * To link, run `node bin/set-up-vs-code.js`
+ * To unlink, run `node bin/set-up-vs-code.js --unlink`
+ */
+
+const fs = require( 'fs' );
+const path = require( 'path' );
+const chalk = require( 'chalk' );
+const yargs = require( 'yargs' );
+
+const settingsPath = path.resolve( __dirname, '../.vscode/settings.json' );
+const settingsSamplePath = path.resolve( __dirname, '../.vscode/settings-sample.jsonc' );
```

Comment:
Linking the entire folder would create less problems for existing workspaces.

```suggestion
const settingsPath = path.resolve( __dirname, '../.vscode' );
const settingsSamplePath = path.resolve( __dirname, '../.vscode-sample' );
```

Reply:
Can you please explain how it would create less problems?

IMHO, it will be more problematic because it will skip the entire directory if it exists and thus won't have `extensions.json` as well. If we want to no override user preferences, then we need to make sure not to add `settings.json`.

Reply:
> it will skip the entire directory if it exists

This is what I was hoping for, but... I think I understand your approach better now too: So long as I don't run the link command I shouldn't see anything overriding my workspace.

Reply:
Correct and there is always `yarn vscode:unlink` in case someone wants to opt-out.